superset-frontend/plugins/plugin-chart-handlebars/src/plugin/controls/pagination.tsx

```diff
@@ -24,23 +24,23 @@ import {
 import { FeatureFlag, isFeatureEnabled, t } from '@superset-ui/core';
 import { PAGE_SIZE_OPTIONS } from '../../consts';
 
-export const serverPaginationControlSetRow: ControlSetRow =
```

Comment:
@michael-s-molina I am not too sure about this change. It seems that server_pagination should be shown by default now as the  `DASHBOARD_NATIVE_FILTERS` is also by default